packages/input-text/src/styledComponents/TextInput.js

```diff
@@ -0,0 +1,85 @@
+import styled, { css } from "styled-components";
+import { colors, fontSizes } from "@crave/farmblocks-theme";
+
+const labelColor = props => {
+  return props.invalid
+    ? colors.STRAWBERRY
+    : props.disabled ? colors.GREY_32 : colors.CARBON;
+};
+const inputBorderColor = props => {
+  return props.invalid ? colors.STRAWBERRY : colors.GREY_16;
+};
+const inputBackgroundColor = props => {
+  return props.disabled ? colors.GREY_16 : "white";
+};
+const inputBoxShadow = focused => props => {
+  if (!focused && (props.disabled || props.filled)) {
+    return css`
+      box-shadow: none;
+    `;
+  }
+  const shadowSize = props.invalid ? "0 2px 2px 0" : "0 4px 4px 0";
+  return css`
+    box-shadow: ${shadowSize} ${colors.GREY_16};
+  `;
+};
+const Container = styled.label`
+  font-family: Lato, sans-serif;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 24px;
+
+  .label {
+    order: 1;
+    text-transform: uppercase;
+    font-size: ${fontSizes.SMALL}px;
+    margin-bottom: 8px;
+    color: ${labelColor};
+  }
```

Comment:
Missing `font-weight`, `line-height` and `letter-spacing`.

![image](https://user-images.githubusercontent.com/17936244/33731875-a8d66fb8-db6b-11e7-9600-ca8d3eca581b.png)

Reply:
oops, good catch

Reply:
font-weight 600 was possible using the title property of the text component, line height and letter spacing I had to set